recipe/build.sh

```diff
@@ -13,22 +13,36 @@ export ESMF_NETCDF="split"
 export ESMF_NETCDF_INCLUDE=${PREFIX}/include
 export ESMF_NETCDF_LIBPATH=${PREFIX}/lib
 
-export ESMF_COMM=mpich3
+if [[ $(uname) == Darwin ]]; then
+  export ESMF_CPP="clang-cpp -P -x c"
+else
+  export ESMF_CPP="${CPP} -E -P -x c"
+fi
+
+if [[ -z "$mpi" || "$mpi" == "nompi" ]]; then
+  export ESMF_F90=${FC}
+  export ESMF_CXX=${CXX}
+else
+  export ESMF_F90=mpif90
+  export ESMF_CXX=mpicxx
+fi
```

Comment:
If these 3 environment variables are defined, no links to `gcc` and `gfortran` should be needed.